### ISSUE TITLE
ts-force-gen: Picklist (Multi-select) generates the type as string instead of string[]

### ISSUE THREAD

**Issue by ScottMGerstl** [ChuckJonas/ts-force#64]
version: 2.6.0
Example:
```
@sField({ 
   apiName: 'Groups__c', 
   createable: true, 
   updateable: true, 
   required: false,
   reference: undefined, 
   childRelationship: false, 
   salesforceType: SalesforceFieldType.MULTIPICKLIST, 
   salesforceLabel: 'Groups', 
   externalId: false 
})
public groups: string;
```

**Comment by ChuckJonas** [comment]
interesting, Looks like it only correctly generates as a `[]` when the `enforcePicklistValues:true` in the config.

Will fix

**Comment by ChuckJonas** [comment]
fixed in v2.6.2
